core/templates/pages/landing-pages/stewards-landing-page/stewards-landing-page.component.html

```diff
@@ -143,7 +143,7 @@ <h2 class="oppia-stewards-landing-h2" style="padding-right: 15px;">Support from
               <button uib-dropdown-toggle class="btn btn-secondary selection dropdown-toggle" type="button" aria-expanded="false" data-toggle="dropdown">
                 <[$ctrl.getActiveTabNameInSingularForm()]>
               </button>
-              <ul uib-dropdown-menu style="width: inherit;">
+              <ul uib-dropdown-menu class="oppia-dropdown-menu">
```

Comment:
See my comment above -- this is a case where the class names clash if you don't include the directive name as a prefix.

Reply:
Added directive name